README.md

```diff
@@ -1724,6 +1725,7 @@ Update Time, five active automations, webhooks.
   * [codepen.io](https://codepen.io/) — CodePen is a playground for the front-end side of the web.
   * [codesandbox.io](https://codesandbox.io/) — Online Playground for React, Vue, Angular, Preact, and more.
   * [Components.studio](https://webcomponents.dev/) - Code components in isolation, visualize them in stories, test them, and publish them on npm.
+  * [Cursor](https://cursor.com/) — An AI-first code editor and IDE (a fork of VS Code) that uses LLMs for code generation, chat, and large-scale refactoring. Free tier includes the full IDE, (limited) AI chat requests (e.g., 50-300/month) with standard models, and limited AI autocomplete.
```

Comment:
unfortunately payment terms aren't clear enough, all I can see is "limited requests" no details of what that means.

Pretty normal problem with heavily AI based services so we dont tend to add them with such unclear pricing